liam2/alignment.py

```diff
@@ -329,15 +368,17 @@ def compute(self, context, score, need, filter=None, take=None, leave=None,
 
         func = self.align_no_link if link is None else self.align_link
         return func(context, score, need, filter, take, leave, expressions,
-                    possible_values, errors, frac_need, link, secondary_axis)
+                    possible_values, errors, frac_need, link, secondary_axis, method=method)

```

Comment:
The rule I use is: if I know I use all arguments in all usages of the function, I don't name arguments explicitly.